### PR TITLE
Test PR : added packages

### PR DESCRIPTION
**Pull request by priyverm**
This PR is a test PR.

### CODE REVIEW

File: tests/ffi/network/test_largepckts_icmp.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+prepare_test
+reload_config
+
+#Here $timper is allowed percentage delay and $ploss is allowed packet loss
+timper=5
+ploss=0
+echo " "
+echo "Creating Containers*"
+echo " "
+podman run -d --name orderly stream9-stress
```
Comment (Yarboa) [inline tests/ffi/network/test_largepckts_icmp.sh:20]: Which registry it is using?

File: tests/ffi/network/test_largepckts_icmp.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+loss=$(cat log | grep packet | tail -1 | awk '{print $6}' | sed 's/%//g')
+echo -e "\e[1;32m Packet loss is $loss percent  \e[0m"
+if [[ $loss -gt $ploss ]]
+then
+  echo -e "\e[1;31m Interference is present - presence of packet loss \e[0m"
+  exit 1
+else
+        echo -e "\e[1;32m Interference is not present - No packet loss \e[0m"
+fi
+./cleanup.sh
```
Comment (Yarboa) [inline tests/ffi/network/test_largepckts_icmp.sh:65]: Which clean up is it?
maybe call for that? 
https://github.com/containers/qm/pull/340/files#diff-1d7a826dde8c6bdf6902d9aa66478b18b0b9a4114f7e378cb0463091bfe2506dR10

**Review by Yarboa — COMMENTED** [review]
Currently test is failing 
https://artifacts.dev.testing-farm.io/1f036671-2a16-4a2d-bbe8-e104d0865125/

One more thing consider using ffi-tools
PTAL https://gitlab.com/CentOS/automotive/container-images/ffi-tools/-/blob/main/Containerfile

**Comment by dougsland** [comment]
nice, looking forward to see the final version of the patch.

File: tests/ffi/network/test_largepckts_icmp.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+disk_cleanup
+prepare_test
+reload_config
+
+#Here $timper is allowed percentage delay and $ploss is allowed packet loss
+timper=5
+ploss=0
+echo " "
+echo "Creating Containers*"
```
Comment (Yarboa) [inline tests/ffi/network/test_largepckts_icmp.sh:18]: Consider using env vars with defaults
like this 

TIMPER=${TIMPER:-5}
the same for ploss

File: tests/ffi/network/main.fmf [hunk 4]
```diff
@@ -0,0 +1,8 @@
+summary: Sending large packets
+test: ./test_largepckts_icmp.sh 'ping -i 0.01 -s 1600 %s'
```
Comment (Yarboa) [inline tests/ffi/network/main.fmf:2]: Consider one of the two
chmod +x ./test_largepckts_icmp.sh
or use this 
/bin/bash  ./test_largepckts_icmp.sh

File: tests/ffi/network/test_largepckts_icmp.sh [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+sleep 2
+echo -e "\e[1;35m Time taken to ping in ideal condition is $idealtime ms \e[0m"
+echo "*****Confusion container creates stress condition, while orderly pings partner 10 times*****"
+echo " "
+podman exec -it confusion sh -c "$STRESS_CMD" &
+sleep 5
+understresstime=$(podman exec -it orderly sh -c "ping -c 10 $partneripadd"  &>> log && cat log | grep packet | tail -1 | awk '{print $NF}'| sed 's/[ms]//g')
+sar -n EICMP 1 5 &
+echo ""
+sleep 10
```
Comment (Yarboa) [inline tests/ffi/network/test_largepckts_icmp.sh:40]: same here, recommended variable to be configures with ENVS

File: tests/ffi/network/test_largepckts_icmp.sh [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+pkill ping
+echo  -e "\e[1;35m Time taken to ping target in while in a heavy traffic is $understresstime ms \e[0m"
+increase=$(awk -v t1="$idealtime" -v t2="$understresstime" 'BEGIN{printf "%.0f", (t2-t1)/t1 * 100}')
+sleep 1
+echo ""
+echo -e "\e[1;32m Percentage increase in time taken is $increase percent \e[0m"
```
Comment (Yarboa) [inline tests/ffi/network/test_largepckts_icmp.sh:46]: Consider using this 
info_message  from https://github.com/containers/qm/blob/main/tests/e2e/lib/utils

File: tests/ffi/network/test_largepckts_icmp.sh [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+prepare_test
+reload_config
+
+#Here $timper is allowed percentage delay and $ploss is allowed packet loss
+timper=5
+ploss=0
+echo " "
+echo "Creating Containers*"
+echo " "
+podman run -d --name orderly stream9-stress tail -f /dev/null
```
Comment (Yarboa) [inline tests/ffi/network/test_largepckts_icmp.sh:20]: I think the tests should run in a different way
U assume that orderly should run with ASIL filesystem space while confusion should be in QM, 
Am i correct?

**Review by Yarboa — COMMENTED** [review]
@priyverm If it helps @pengshanyu  and myself could contribute into it,
You need to enable us as contributors to your forked repo

**Comment by dougsland** [comment]
nice one, still working on it?

**Pull request closed without merging**